src/dtx/dtx_resync.c

```diff
@@ -245,6 +245,13 @@ dtx_status_handle_one(struct ds_cont_child *cont, struct dtx_entry *dte, daos_un
 		 * let's commit the DTX globally.
 		 */
 		D_GOTO(out, rc = DSHR_NEED_COMMIT);
+	case -DER_OOG:
+	case -DER_HG:
+		D_WARN("Need retry resync for DTX " DF_DTI " because of " DF_RC "\n",
+		       DP_DTI(&dte->dte_xid), DP_RC(rc));
+		/* Yield to give more chance for network recovery. */
+		ABT_thread_yield();
```

Comment:
please use dss_sleep() for dss scheduler purpose.

Reply:
> please use dss_sleep() for dss scheduler purpose.

DTX resync may block the other IO process, it is better to make it to be scheduled ASAP.